op-chain-ops/genesis/config.go

```diff
@@ -1134,6 +1144,34 @@ type L1Deployments struct {
 	DataAvailabilityChallengeProxy    common.Address `json:"DataAvailabilityChallengeProxy"`
 }
 
+func CreateL1DeploymentsFromContracts(contracts *addresses.L1Contracts) *L1Deployments {
```

Comment:
Will replace `L1Deployments` struct with standard addresses struct(s) in a future pr . But needed to add this translation helper function for now